src/video-player.js

```diff
@@ -533,9 +533,8 @@ class VideoPlayer extends Utils.mixin(Eventable) {
       return this.videojs.cloudinary.source(publicId, options);
     }
 
-    // Interactive plugin - available in full (not light) build only
-    if (this.videojs.interactive) {
-      this.videojs.interactive(this.videojs, options);
```

Comment:
why are we removing these 2 lines?

Reply:
The "interactive" plugin was supposed to wrap the shoppable plugin as well as other relevant features, this never happened and it only contains the shoppable plugin.
Meanwhile, the interaction-area plugin was introduced, so this just became even more confusing.
Bottom line - the "interactive" plugin became "shoppable"